cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -188,8 +198,23 @@ instance FromJSON PartialNodeConfiguration where
            , pncValidateDB = mempty
            , pncShutdownIPC = mempty
            , pncShutdownOnSlotSynced = mempty
+           , pncMaybeMempoolCapacityOverride
            }
     where
+      parseMempoolCapacityBytesOverride :: Object -> J.Parser (Maybe MempoolCapacityBytesOverride)
+      parseMempoolCapacityBytesOverride v = do
+        maybeBytes <- v .:? "MempoolCapacityBytesOverride"
+        case maybeBytes of
+          Just bytes -> return (Just (MempoolCapacityBytesOverride (MempoolCapacityBytes bytes)))
```

Comment:
This could be made more concise with the following pattern match:
```
case maybeBytes of
  Just (Aeson.String "NoOverride") -> ...
  Just (Aeson.Number bytes) -> ...
  Just wrongAesonType -> ...
  Nothing -> ...
```